.clang-format

```diff
@@ -86,7 +86,7 @@ PenaltyIndentedWhitespace: 0
 PointerAlignment: Right
 
 QualifierAlignment: Custom
-QualifierOrder: ['inline', 'static', 'restrict', 'type', 'const', 'volatile' ]
+QualifierOrder: [ 'inline', 'static', 'const', 'volatile', 'restrict', 'type' ]
```

Comment:
Why the change in order?

Reply:
Mainly because of type being the right-most. Such that we do not have something like `int const *`, but instead `const int *`. The rest is somewhat arbitrary/up for debate.